src/cp_control_types.F

```diff
@@ -38,7 +38,24 @@ MODULE cp_control_types
       INTEGER, DIMENSION(2)                :: distribution_layout
       INTEGER                              :: blocked
    END TYPE pw_grid_option
+! **************************************************************************************************
+! \brief parameters for EMD/RTP calculations involving MO projections
+! **************************************************************************************************
+   TYPE proj_mo_type
```

Comment:
Please add default initializes to this type: https://www.cp2k.org/conv#c016.

Reply:
I Have initialized the values, expect for the allocatable ones. Is it done properly?